scripts/install-devnet.sh

```diff
@@ -2,6 +2,9 @@
 
 set -eu
 
-pip3 install "starknet-devnet==$STARKNET_DEVNET"
+# Get default from config.json file
+STARKNET_DEVNET_DEFAULT=$(node -e "console.log(require('../config.json').STARKNET_DEVNET)")
+
+pip3 install "starknet-devnet==${STARKNET_DEVNET:=$STARKNET_DEVNET_DEFAULT}" | { grep --invert-match 'Requirement already satisfied:' || :; }
```

Comment:
1. Uses STARKNET_DEVNET_DEFAULT fallback
2. grep removes `Requirement already satisfied` logs.
3. `|| :;` part, which is like `or nothing;`, prevents grep from `exit`ing the process.